config/config.go

```diff
@@ -53,6 +53,42 @@ type SourcesApiConfig struct {
 	TenantTranslatorUrl       string
 }
 
+//String() returns a string that shows the settings in which the pod is running in
+func (s SourcesApiConfig) String() string {
+	var b strings.Builder
+	fmt.Fprintf(&b, "%s: %s\n", "AppName", parsedConfig.AppName)
+	fmt.Fprintf(&b, "%s: %v\n", "Hostname", parsedConfig.Hostname)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaBrokers", parsedConfig.KafkaBrokers)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaTopics", parsedConfig.KafkaTopics)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaGroupID", parsedConfig.KafkaGroupID)
+	fmt.Fprintf(&b, "%s: %v\n", "MetricsPort", parsedConfig.MetricsPort)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevel", parsedConfig.LogLevel)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevelForMiddlewareLogs", parsedConfig.LogLevelForMiddlewareLogs)
+	fmt.Fprintf(&b, "%s: %v\n", "LogGroup", parsedConfig.LogGroup)
+	fmt.Fprintf(&b, "%s: %v\n", "LogHandler", parsedConfig.LogHandler)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevelForSqlLogs", parsedConfig.LogLevelForSqlLogs)
+	fmt.Fprintf(&b, "%s: %v\n", "MarketplaceHost", parsedConfig.MarketplaceHost)
+	fmt.Fprintf(&b, "%s: %v\n", "AwsRegion", parsedConfig.AwsRegion)
+	fmt.Fprintf(&b, "%s: %v\n", "DatabaseHost", parsedConfig.DatabaseHost)
+	fmt.Fprintf(&b, "%s: %v\n", "DatabasePort", parsedConfig.DatabasePort)
+	fmt.Fprintf(&b, "%s: %v\n", "DatabaseName", parsedConfig.DatabaseName)
+	fmt.Fprintf(&b, "%s: %v\n", "FeatureFlagsEnvironment", parsedConfig.FeatureFlagsEnvironment)
+	fmt.Fprintf(&b, "%s: %v\n", "FeatureFlagsUrl", parsedConfig.FeatureFlagsUrl)
+	fmt.Fprintf(&b, "%s: %v\n", "FeatureFlagsService", parsedConfig.FeatureFlagsService)
+	fmt.Fprintf(&b, "%s: %v\n", "CacheHost", parsedConfig.CacheHost)
+	fmt.Fprintf(&b, "%s: %v\n", "CachePort", parsedConfig.CachePort)
+	fmt.Fprintf(&b, "%s: %v\n", "SlowSQLThreshold", parsedConfig.SlowSQLThreshold)
+	fmt.Fprintf(&b, "%s: %v\n", "Psks", parsedConfig.Psks)
+	fmt.Fprintf(&b, "%s: %v\n", "BypassRbac", parsedConfig.BypassRbac)
+	fmt.Fprintf(&b, "%s: %v\n", "StatusListener", parsedConfig.StatusListener)
+	fmt.Fprintf(&b, "%s: %v\n", "BackgroundWorker", parsedConfig.BackgroundWorker)
+	fmt.Fprintf(&b, "%s: %v\n", "Migrations", parsedConfig.MigrationsSetup)
+	fmt.Fprintf(&b, "%s: %v\n", "MigrationsReset", parsedConfig.MigrationsReset)
```

Comment:
I also don't think we need these flags, the first 2 would only be when specifying backgroundworker/listener - and the other two are kind of internal to the migrate/reset flags on the app.